Models/Entities/SalesOrderItem.cs

```diff
@@ -13,10 +14,11 @@ public class SalesOrderItem : _Base
         public double? UnitPrice { get; set; } = 0;
         public double? Quantity { get; set; } = 1;
         public double? Total { get; set; } = 0;
+        public string? FileImage { get; set; }
```

Comment:
Should it be possible list all the images in the frontend directly based on the ProductId? Or this field on this class is necesssary